src/main/java/org/folio/service/inventory/InventoryManager.java

```diff
@@ -1290,6 +1300,25 @@ private Future<List<JsonObject>> searchStorageExistingItems(String poLineId, Str
   }
 
   private Future<List<JsonObject>> fetchHoldingsByFundIds(List<String> holdingIds, RequestContext requestContext) {
```

Comment:
interesting why it named as fetchHoldingsByFundIds instead of fetchHoldingsByHoldingIds, propose to rename

Reply:
good catch. I didn't notice typo. Thank you